test/zeroconf/node-test.py

```diff
@@ -3,7 +3,7 @@
 import subprocess
 
 def dig_for(hostname):
-    proc = subprocess.run(['dig', hostname, '@localhost', '+short'], check=False, stderr=subprocess.STDOUT, stdout=subprocess.PIPE)
+    proc = subprocess.run(['dig', hostname, '@localhost', '+short'], check=True, stderr=subprocess.STDOUT, stdout=subprocess.PIPE)
```

Comment:
Isn't `check=True` the default?

Reply:
No, it's not. https://docs.python.org/3/library/subprocess.html#subprocess.run

Reply:
:+1: